build.cmd

```diff
@@ -6,8 +6,18 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly. 
 
-echo %2 | find /I "outerloop"
-if "%errorlevel%" equ "0" (
+set outloop=false
+
+REM this is a temporary step until the build tools provide a proper hook. 
+REM it will need to deal with multiple include and exclude categories.
+REM See dotnet/corefx#1477
+
+echo %* | findstr /i /C:"/p:WithCategories=OuterLoop"  1>nul
```

Comment:
/p:Withcategories=Custom,OuterLoop is invalid for msbuild and current build tool. i.e current build tool does not support such syntaxes. You need to do /p:Withcategories=Custom /p:WithcategoriesOuterLoop